### DISCUSSION TITLE
no data get

### DISCUSSION THREAD

**Discussion by karelbyte** [spatie/laravel-medialibrary#1941] [Q&A]
Hello friends, I am trying to use this tool but apparently I have some detail that does not let me work
I have followed the installation steps and it seems everything is correct
table "media" has been created
my model has complained like this

<? php
namespace App \ Models;
use Illuminate \ Database \ Eloquent \ Model;
use Spatie \ MediaLibrary \ HasMedia;
use Spatie \ MediaLibrary \ InteractsWithMedia;
class Product extends Model implements HasMedia
{
    use InteractsWithMedia;
    protected $ fillable = [
        'id', 'name', 'slug', 'description', 'price', 'status'
    ];
}
 
In my controller I have added an image from which I have combined that it arrives without problems
and it's saved in public using this
$ product-> addMedia ($ request-> file ('media')) -> toMediaCollection ();

if I check the product and a half table, the data is apparently all good
 $ product = Product :: query () -> find (7); // ID CHECK
        $ media = $ product-> getMedia ();
        return response ($ average);
but getmedia returns nothing
some help that I'm doing wrong

**Comment by divian86** [comment]
Hi. I have the same issue, how did you make it work?
